src/components/ForgotPassword.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import reset from "../assets/Icons/Light Mode Icon/reset.png";
+import Image from "next/image";
+const ForgotPassword = () => {
+  return (
+    <div className="w-full flex flex-col justify-center items-start bg-white rounded-3xl shadow-lg">
+      <span>
```

Comment:
Please utilize a div here and use a `flex` and `justify between` and `w-full` here